application/src/main/java/uk/gov/hmcts/ccd/definition/store/AuthCheckerConfiguration.java

```diff
@@ -22,7 +22,7 @@
     protected static final String ROLE_CCD_IMPORT = "ccd-import";
     private static final String REGEX_URI_IMPORT = "^" + ImportController.URI_IMPORT  + "/?$";
 
-    @Value("#{'${casedefinitionstore.authorised.services}'.split(',')}")
+    @Value("#{'${idam.s2s-authorised.services}'.split(',')}")
```

Comment:
remove whole bean .. now s2s validation happening in the serviceAuthFilter

Reply:
what do you mean by whole bean.. do you mean AuthCheckerConfiguration

Reply:
`authorizedServicesExtractor` bean